2-mandatory/4-stocks.js

```diff
@@ -48,7 +60,17 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
 function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
+  // TODO
+  let priceChange = [];
+  for (i = 0; i < closingPricesForAllStocks.length; i++) {
+    priceChange.push(
+      Math.round(
+        (closingPricesForAllStocks[i][4] - closingPricesForAllStocks[i][0]) *
```

Comment:
Could you calculate the difference dynamically (i.e. without had coding the element for the last day's price?)